src/components/Mapper/Mapper.jsx

```diff
@@ -69,6 +78,10 @@ export default function Mapper() {
     setEncounterList(setAllEncounters(locationName))
   }, [swarm, radar, tod, incense, surfIncense, rod])
 
+  useEffect(() => {
+    setLocationList(getRoutesFromPokemonId(getPokemonIdFromName(pokemon)))
+  }, [pokemon])
+
   const handleTODChange = (event, nextView) => {
```

Comment:
Acronyms are evil

Reply:
Changed every instance to TimeOfDay or timeOfDay